firstvoices/backend/serializers/fields.py

```diff
@@ -118,7 +120,10 @@ def to_representation(self, data):
         """
         Transform the *outgoing* native value into primitive data.
         """
-        return [{"text": item} if item is not None else None for item in data]
+        return [
```

Comment:
Currently this change adds arbitrary uuid4's to all text list fields, however they are not consistent. I have tested and re-queriying the API will mean these fields have ids but they will not be the same. I can make adjustments if this would cause errors in formatting for the FE.